MainModule/Server/Core/Logs.lua

```diff
@@ -207,6 +213,22 @@ return function(Vargs)
 				return tab
 			end;
 			
+			ReplicationLogs = function()
+				local tab = {}
+				for i,v in pairs(Logs.Replications) do
+					table.insert(tab,{Text=v.Player.." "..v.Action.." "..v.ClassName;Desc = v.Path})
+				end
+				return tab
+			end;
+			
+			NetworkOwners = function()
+				local tab = {}
+				for i,v in pairs(Logs.NetworkOwners) do
+					table.insert(tab,{Text = tostring(v.Player).." made "..tostring(v.Part),Desc = v.Path})
+				end
+				return tab
+			end;
+			
```

Comment:
Do not add replications and network owners. They were removed in the removal of FD spesific code.